test/integration_test_helper.rb

```diff
@@ -20,16 +20,8 @@
 end
 
 Selenium::WebDriver::Chrome::Service.driver_path = ENV['TESTDRIVER_PATH'] || Foreman::Util.which('chromedriver', Rails.root.join('node_modules', '.bin'))
-Capybara.register_driver :selenium_chrome do |app|
-  options = Selenium::WebDriver::Chrome::Options.new
-  options.args << '--disable-gpu'
-  options.args << '--no-sandbox'
-  options.args << '--window-size=1024,768'
-  options.args << '--headless' unless ENV['DEBUG_JS_TEST'] == '1'
-  Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
-end
 Capybara.configure do |config|
-  config.javascript_driver      = ENV["JS_TEST_DRIVER"]&.to_sym || :selenium_chrome
+  config.javascript_driver      = ENV.fetch("JS_TEST_DRIVER") { ENV['DEBUG_JS_TEST'] ? :selenium_chrome : :selenium_chrome_headless }.to_sym
```

Comment:
We should probably also adapt this line https://github.com/theforeman/foreman/blob/develop/test/integration_test_helper.rb#L302

When not set to the correct `JS_TEST_DRIVER` (`:selnium_chorme`,  `selenium_chrome_headless`, `selenium_chrome_remote`) it throws the following error:

```
ShowMeTheCookies::SeleniumSiteNotVisitedError: Can't set a cookie on about:blank. Visit a url in your app first.
    test/integration_test_helper.rb:330:in `set_request_user'
    test/integration_test_helper.rb:321:in `login_admin'
    test/test_helper.rb:73:in `before_setup'
    test/integration_test_helper.rb:348:in `before_setup'
```

Maybe the if clause in L302 can even be removed?